native/cocos/bindings/manual/jsb_module_register.cpp

```diff
@@ -160,7 +165,7 @@ bool jsb_register_all_modules() {
 
 #endif // USE_MIDDLEWARE
 
-#if USE_PHYSICS_PHYSX
+#if USE_PHYSICS_PHYSX && CC_PLATFORM != CC_PLATFORM_OPENHARMONY
```

Comment:
Can not support physx on open harmony or just haven't implemented it?

Reply:
Not yet implemented

Reply:
Is there an issue for it and other codes that needs to adapt later?

Reply:
It is mainly caused by some third-party libraries or incomplete adaptation. After the adaptation is completed, it can be removed here.